frontend/src/metabase/entities/schemas.js

```diff
@@ -16,19 +15,18 @@ import {
 
 // This is a weird entity because we don't have actual schema objects
 
-const listDatabaseSchemas = GET("/api/database/:dbId/schemas");
-const getSchemaTables = GET("/api/database/:dbId/schema/:schemaName");
-const getVirtualDatasetTables = GET("/api/database/:dbId/datasets/:schemaName");
```

Comment:
moved these to metabase/services to follow the pattern of definining all API endpoints there.